### ISSUE TITLE
less-loader support please

### ISSUE THREAD

**Issue by partoneplay** [gsoft-inc/craco#273]
The `craco-less` `craco-antd` project is almost dead.

I read doc [less-loader](https://www.npmjs.com/package/less-loader#lessoptions) and my `craco.config.js`：

```js
module.exports = {
  webpack: {
    alias: alias,
    configure: {
      module: {
        rules: [{
          test: /\.less$/,
          use: [{
            loader: 'less-loader', // compile Less to CSS
            options: {
              lessOptions: {
                modifyVars: {
                  // antd customize
                  'primary-color': '#038952',
                  'layout-header-background': '#0B7E4F',
                  'layout-sider-background': '#ffffff',
                },
                javascriptEnabled: true,
              },
            },
          }],
        }],
      },
    },
  },
};
```

But it does't work.

I must use `craco@^6.0.0`

**Comment by wangpeng1994** [comment]
Below article might be help?~

https://github.com/xtech-guru/create-react-app/blob/add-less-support/docusaurus/docs/adding-a-less-stylesheet.md

**Comment by partoneplay** [comment]
> Below article might be help?~
> 
> https://github.com/xtech-guru/create-react-app/blob/add-less-support/docusaurus/docs/adding-a-less-stylesheet.md

>Note: this feature is available with react-scripts@4.1.0 and higher.

The current latest version is 4.0.3. So I have to wait for official update.
